inception-recommendation/src/main/java/de/tudarmstadt/ukp/inception/recommendation/chart/resources/chart.js

```diff
@@ -0,0 +1,48 @@
+/*
+#Copyright 2019
+#Ubiquitous Knowledge Processing (UKP) Lab
+#Technische Universität Darmstadt
+#
+#Licensed under the Apache License, Version 2.0 (the "License");
+#you may not use this file except in compliance with the License.
+#You may obtain a copy of the License at
+# 
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+#Unless required by applicable law or agreed to in writing, software
+#distributed under the License is distributed on an "AS IS" BASIS,
+#WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#See the License for the specific language governing permissions and
+#limitations under the License.
+*/
+
+function OnSuccess(a) {
```

Comment:
Can you please add some documentation here? What is this doing?

Function names should start with small letters. Since this function is added to the global JS namespace of the page, a more speaking name would seem to be reasonable - and a more unambiguous one with less risk of a collision?

Reply:
Good idea.